doc/install/mne_python.rst

```diff
@@ -19,64 +21,104 @@ MNE-Python 0.18 only supports Python version 3.5 or higher. We strongly
 recommend the `Anaconda`_ distribution of Python, which comes with more than
 250 scientific packages pre-bundled, and includes the ``conda`` command line
 tool for installing new packages and managing different package sets
-("environments") for different projects. Follow the `installation instructions
-for Anaconda`_; when you are done, you should see a similar output if you type
-the following command in a terminal:
+("environments") for different projects.
+
+To get started, follow the `installation instructions for Anaconda`_.
+When you are done, if you type the following commands in a ``bash`` terminal,
+you should see outputs similar to the following (assuming you installed
+conda to ``/home/user/anaconda3``)::
+
+    $ conda --version && python --version
+    conda 4.6.2
+    Python 3.6.7 :: Anaconda, Inc.
+    $ which python
+    /home/user/anaconda3/bin/python
+    $ which pip
+    /home/user/anaconda3/bin/pip
+
+.. collapse:: |hand-stop-o| If you get an error or these look incorrect...
+    :class: danger
+
+    **If you see something like**::
+
+        conda: command not found
+
+    It means that your ``PATH`` variable (what the system uses to find
+    programs) is not set properly. In a correct installation, doing::
```

Comment:
sorry, late to the party. There is another possibility and something I've observed quite a bit. People install anaconda using one shell and their default is another shell. So while they have it installed, they are just in the wrong shell.

Reply:
cool. wanna make a new PR to add that @jasmainak ?

Reply:
see: https://github.com/mne-tools/mne-python/pull/6404 :)